assignment3.rb

```diff
@@ -0,0 +1,143 @@
+require 'csv'
+
+class Employee
+  attr_reader :name, :age, :department, :division, :designation
+
+  def initialize(name, age, department, division, designation)
+    @name = name
+    @age = age
+    @department = department
+    @division = division
+    @designation = designation
+  end
+
+  def to_csv_row
+    [name, age, department, division, designation]
+  end
+end
+
+# Subclasses for different divisions
+class DevelopmentEmployee < Employee; end
```

Comment:
please remove Employee in child classes.

Reply:
removed employee